app/controllers/users.js

```diff
@@ -1,6 +1,8 @@
 const validateUser = require('../services/help/userValidate'),
+  config = require('../../config'),
   bcrypt = require('bcryptjs'),
-  salt = bcrypt.genSaltSync(10),
+  { magicNumberTen } = config.common.bcrypt,
```

Comment:
`magicNumberTen` is a bad name for a variable, something like saltNumber could be

Reply:
ok!